test-specifications/token-service/tokenCreateTransaction.md

```diff
@@ -0,0 +1,1158 @@
+| 3       | Creates a token with a memo that is the maximum length      | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, memo="This is a really long memo but it is still valid because it is 100 characters exactly on the money!!"  | The token creation succeeds and the token's memo equals "This is a really long memo but it is still valid because it is 100 characters exactly on the money!!". | N                 |
+| 4       | Creates a token with a memo that exceeds the maximum length | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, memo="This is a long memo that is not valid because it exceeds 100 characters and it should fail the test!!" | The token creation fails with an MEMO_TOO_LONG response code from the network.                                                                                  | N                 |
+
+#### JSON Request Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 12,
+  "method": "createToken",
+  "params": {
+    "name": "testname",
+    "symbol": "testsymbol",
+    "treasuryAccountId": "0.0.2",
+    "memo": "testmemo"
+  }
+}
+```
+
+#### JSON Response Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 12,
+  "result": {
+    "tokenId": "0.0.541",
+    "status": "SUCCESS"
+  }
+}
+```
+
+### **Token Type:**
+
+- The type of token to be created.
+
+| Test no | Name                                | Input                                                                                                                 | Expected response                                                                        | Implemented (Y/N) |
+|---------|-------------------------------------|-----------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------|-------------------|
+| 1       | Creates a fungible token            | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, tokenType="ft"                         | The token creation succeeds and the token is a fungible token.                           | N                 |
+| 2       | Creates an NFT                      | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, supplyKey=<VALID_KEY>, tokenType="nft" | The token creation succeeds and the token is an NFT.                                     | N                 |
+| 3       | Creates an NFT without a supply key | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, tokenType="nft"                        | The token creation fails with an TOKEN_HAS_NO_SUPPLY_KEY response code from the network. | N                 |
```

Comment:
It's this the same as `Creates an NFT with an initial supply of zero without a supply key`

Reply:
Also, we could add `Creates a fungible token without a supply key` test case

Reply:
> It's this the same as `Creates an NFT with an initial supply of zero without a supply key`

Didn't realize the redundancy here, removed here and in #228 as well.

> Also, we could add `Creates a fungible token without a supply key` test case

I believe most, if not all, fungible tokens created in these tests are created without supply keys. Is there something more specific you would want this to test?